src/orange/detail/RectArrayInserter.hh

```diff
@@ -0,0 +1,46 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2022-2023 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file orange/detail/RectArrayInserter.hh
+//---------------------------------------------------------------------------//
+#pragma once
+
+#include <vector>
```

Comment:
IWYU (I need to get someone's help to set up an automatic IWYU/clang-format bot...)